xgbserver/rockcraft.yaml

```diff
@@ -0,0 +1,78 @@
+# Based on https://github.com/kserve/kserve/blob/master/python/xgb.Dockerfile
+# 
+# See ../CONTRIBUTING.md for more details about the patterns used in this rock. 
+# This rock is implemented with some atypical patterns due to the native of the upstream
+# Dockerfile.
+name: xgbserver
+summary: xgboost server for Kserve deployments
+description: "Kserve xgboost server"
+version: "0.11.2"
+license: Apache-2.0
+base: ubuntu@22.04
+platforms:
+    amd64:
+run-user: _daemon_
+services:
+  xgbserver:
+    override: replace
+    summary: "xgbserver server service"
+    startup: enabled
+    command: "python -m xgbserver [ ]"
+entrypoint-service: xgbserver
+
+parts:
+  security-team-requirement:
+    plugin: nil
+    override-build: |
+      mkdir -p ${CRAFT_PART_INSTALL}/usr/share/rocks
+      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && \
+      dpkg-query --root=${CRAFT_PROJECT_DIR}/../bundles/ubuntu-22.04/rootfs/ -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) \
+      > ${CRAFT_PART_INSTALL}/usr/share/rocks/dpkg.query
+
+  python:
+    plugin: nil
+    source: https://github.com/kserve/kserve.git
+    source-subdir: python
+    source-tag: v0.11.2
+    build-packages:
+      - libgomp1
```

Comment:
I see there is no `build-essential` package compared to other similar ROCKs, although it is being used in the upstream [dockerfile](https://github.com/kserve/kserve/blob/release-0.11/python/xgb.Dockerfile#L12). My guess is that it's okay without it too and I don't see it as a blocker, since the ROCK works as expected. But let's verify that this is intentional.